docs/wiki/feature/irlight.md

```diff
@@ -0,0 +1,45 @@
+---
+layout: wiki
+title: IR Lights
+component: irlight
+description: IR flashlights and multi function lasers.
+group: feature
+category: equipment
+parent: wiki
+mod: ace
+version:
+  major: 3
+  minor: 16
+  patch: 0
+---
+
+## 1. Overview
+
+### 1.1 DBAL-A3
+The DBAL-A3 is a multi function laser and can be run in the following modes:
+
+- IR laser and IR illuminator
+- IR laser
+- IR illuminator
+- Visible laser
+
+### 1.2 SPIR
+The SPIR is an IR only flashlight. It uses a diffused laser, which improves its effective range considerably compared to a conventional flashlight. It has three modes with different beam widths to allow both easy terrain navigation and engagements at longer ranges during difficult lighting conditions.
+
+### 1.3 Long Range Variants
+Due to Arma 3's dated lighting system, flashlights can shine through objects and walls. To avoid these visual bugs, we have opted to tone down their range considerably compared to their real life counterparts. Although the DBAL-A3 has a range of up to 4000 m according to its manufacturer, such intense lights are simply not feasible in Arma.
```

Comment:
```suggestion

Due to Arma 3's dated lighting system, flashlights can shine through objects and walls. To avoid these visual bugs, we have opted to tone down their range considerably compared to their real life counterparts. Although the DBAL-A3 has a range of up to 4000 m according to its manufacturer, such intense lights are simply not feasible in Arma.
```